book/animations.md

```diff
@@ -755,7 +742,7 @@ class Browser:
 ```
 
 Inside `raster`, the composited layer needs to allocate a surface to raster
-itself into; to make this surface requires knowing how big it is. That's
+itself into; this requires knowing how big it is. That's
```

Comment:
Not a significant improvement?

Reply:
I though weird phrasing, but def not critical.